src/helpers.ts

```diff
@@ -4,18 +4,22 @@ import { IndexEquityInfo } from './interface'
 const nseIndia = new NseIndia()
 
 export const getGainersAndLosersByIndex = async (indexSymbol: string) => {
```

Comment:
Make this 'indexSymbol' param as optional. So that user need not pass 'all' as index symbol. If 'indexSymbol' is undefined consider it as 'all'